btr-web/btr-main-app/tests/services/expected-ouptuts.ts

```diff
@@ -34,7 +34,7 @@ export const OwnershipOrControlStatements = {
             minimum: 50
           },
           type: 'shareholding',
-          connectedIndividuals: [{ uuid: '002', legalName: 'Another test name' }]
+          connectedIndividuals: []
```

Comment:
Do we have a test case that makes sure the connectedIndividual is in the submission ?

Reply:
Added, thanks, will merge after all checks pass.